spinnaker/api/application.go

```diff
@@ -13,7 +13,7 @@ import (
 func GetApplication(client *gate.GatewayClient, applicationName string, dest interface{}) error {
 	app, resp, err := client.ApplicationControllerApi.GetApplicationUsingGET(client.Context, applicationName, map[string]interface{}{})
 	if resp != nil {
-		if resp.StatusCode == http.StatusNotFound {
+		if resp != nil && resp.StatusCode == http.StatusNotFound {
```

Comment:
if resp == nil then the next else will fail, right?

Reply:
Never mind, because we're in ```if resp != nil```.  But since we _are_ in ```resp != nil```, this change is unnecessary?